docs/user_guide/scripts/02_medrecord_intro.py

```diff
@@ -0,0 +1,81 @@
+import pandas as pd
+
+import medmodels as mm
+
+# Patients DataFrame (Nodes)
+patients = pd.DataFrame(
+    [
+        ["Patient 01", 72, "M", "USA"],
+        ["Patient 02", 74, "M", "USA"],
+        ["Patient 03", 64, "F", "GER"],
+    ],
+    columns=["ID", "Age", "Sex", "Loc"],
+)
+
+# Medications DataFrame (Nodes)
+medications = pd.DataFrame(
+    [["Med 01", "Insulin"], ["Med 02", "Warfarin"]], columns=["ID", "Name"]
+)
+
+# Patients-Medication Relation (Edges)
+pat_med = pd.DataFrame(
```

Comment:
```suggestion
patients_medications = pd.DataFrame(
```

Better not to have abbreviations in our variable names, I think we should stick to this in the user guide too.